reana_server/config.py

```diff
@@ -94,7 +94,7 @@ def _(x):
 
 #: In production use the following configuration plus adding  the hostname/ip
 #: of the reverse proxy in front of REANA-Server.
-# APP_ALLOWED_HOSTS = [REANA_URL]
+APP_ALLOWED_HOSTS = [REANA_URL]
```

Comment:
If we proceed this way, and the `reana-cluster.yaml` doesn't have a `reana_url` the configuration variable (always the case when [running dev mode](https://github.com/reanahub/reana-cluster/blob/master/reana_cluster/configurations/reana-cluster-minikube-dev.yaml)), `REANA_URL` will fall back to `reana.io` (see [here](https://github.com/reanahub/reana-server/blob/84d06c61c17cbb1a402a09a644220aa3ad88a2b9/reana_server/config.py#L33)) which won't allow requests of any kind to come to the local REANA cluster unless they have the `reana.io` in the host header.

What do you think of doing: (1) do not fall back to `reana.io` but to `None` (2) set `APP_ALLOWED_HOSTS` only if `REANA_URL` is set (for example, we set it when we deploy in production or when we want to [set the GitLab integration](https://github.com/reanahub/reana-ui/pull/40/files#diff-2b2f3fdab1b53ff84ce39f48ddf75281R126)).

Reply:
Oh indeed, I didn't notice that. Changed it, is this what you mean?